iceoryx2/src/service/static_config/message_type_details.rs

```diff
@@ -33,9 +34,9 @@ pub struct TypeDetail {
     pub variant: TypeVariant,
     /// Contains the output of [`core::any::type_name()`].
     pub type_name: String,
-    /// The size of the underlying type.
+    /// The size of the underlying type calculated by [`core::mem::size_of`].
     pub size: usize,
-    /// The alignment of the underlying type.
+    /// The ABI-required minimum alignment of the underlying type calculated by [`core::mem::align_of`].
```

Comment:
Just for completeness, the user is allowed to override this and to set a larger alignment with:

```rust
 let service = node
     .service_builder(&"My/Funk/ServiceName".try_into()?)
     .publish_subscribe::<TransmissionData>()
     .payload_alignment(256)
```

The idea is that the allocator then provides memory that can also be used for SIMD use cases where the input data often requires a larger alignment.